package.json

```diff
@@ -33,13 +43,19 @@
     "safe-buffer": "^5.2.1"
   },
   "devDependencies": {
-    "chai": "^5.0.3",
+    "@types/chai": "latest",
+    "@types/content-type": "^1.1.8",
+    "@types/mocha": "latest",
+    "@typescript-eslint/eslint-plugin": "^6.18.1",
+    "@typescript-eslint/parser": "^6.14.0",
+    "chai": "^5.1.0",
     "eslint": "^8.56.0",
     "eslint-config-standard": "^17.1.0",
     "eslint-plugin-import": "^2.29.1",
     "eslint-plugin-n": "^16.6.2",
     "eslint-plugin-promise": "^6.1.1",
-    "mocha": "^10.3.0"
+    "mocha": "^10.4.0",
+    "typescript": "^5.4.4"
```

Comment:
Updating deps (mocha, chai) can be in the non-TS PR too.

Reply:
also this was related to some ide automations sorry, i disabled completely those function for this repo, will never happen again